README.md

```diff
@@ -111,3 +111,38 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+Place the tool somewhere in your path, using either:
+* make install
```

Comment:
Using their `sudo make install` I still got permission errors, until I did `sudo chmod 755 /usr/local/bin/git-secrets`. (I couldn't install without sudo; I got `cp: cannot create regular file ‘/usr/local/bin/git-secrets’: Permission denied`.)

I think it's OK not to auto-setup, but it'd be useful to at least have instructions for Linux, which I think should go:

```shell
git clone https://github.com/awslabs/git-secrets.git
cd git-secrets
sudo make install
sudo chmod 755 /usr/local/bin/git-secrets
```

and then a "See README for setup instructions if this fails" in setup_git_secrets or wherever is closest to the failure point if you haven't installed it yet.